src/openfermion/measurements/_bayesian_estimators.py

```diff
@@ -23,9 +23,21 @@
 
 class FourierProbabilityDist(object):
     """
-    Stores a estimate of multiple phases in the form arXiv:1809.09697 App.C.
-    In particular, stores the Fourier representation as a sum of sine and
-    cosine waves (Eq. (C1)), and provides methods for manipulation of these.
+    Stores a multivariant Fourier representation of a function linear
+    in its dependent variables.
```

Comment:
This class is only meant to represent linear functions? In principle it can represent any periodic function, right?

Reply:
I think you mean that the representation is linear, and not the function; there should be a better way to word this.

Reply:
Yeah, I mean it's linear in the sense that it's a sum of a periodic function in each phi_j. Let me think about how to describe this better.

Reply:
I think just 'periodic function' works well; the exact definition is one line below, so I don't need to be so descriptive here.